src/components/organisms/sidebar/sidebar.tsx

```diff
@@ -230,6 +241,11 @@ export const Sidebar = () => {
 							</Popover>
 						) : null}
 					</div>
+					<UserFeedbackForm
+						className="absolute bottom-0 left-20"
+						isOpen={isFeedbackOpen}
+						onClose={() => setIsFeedbackOpen(false)}
+					/>
```

Comment:
Opened a ticket to move it into the UserMenu when we will get rid of `featureFlags.enableNewOrgsAndUsersDesign`:
https://linear.app/autokitteh/issue/UI-1151/when-enableneworgsandusersdesign-feature-flag-removed-move-feedback